docs/contributing/testing.md

```diff
@@ -109,10 +109,12 @@ When you create the pull request for your contributions, list what you’ve test
 
 If your contribution changes how a component looks, include before and after screenshots if you can.
 
-### If GitHub shows the build failed
+### If GitHub shows that some checks were not successful
 
-1. At the bottom of your pull request in GitHub, find the line that says **Build Failed**.
-2. Select **Details**.
-3. Under **Build Failed**, select **The build**.
-4. Wait for the build log to appear.
-5. Use the information in the build log to find the problem and fix your code.
+At the bottom of your pull request in GitHub, find the checks that are marked as **Failing**.
+
+For each failing check:
+
+1. Select **Details**.
+2. Wait for the build log to appear.
```

Comment:
Is it always the same amount of time a user waits? Or could we write something like "Wait a few moments for the build log to appear" / "Wait about 10 seconds for the build log to appear"?

Reply:
Good question. It actually appears pretty much straight away so I do wonder if we need a step that says 'Wait for the thing to appear'.

Maybe this could just be

> For each failing check, select **Details** and use the build log to find the problem and fix your code.

Or even we could consider removing this section entirely? I think it used to be helpful because there was an extra step that was not obvious, but now that extra step has been removed maybe the process is simple enough that this is redundant?

@vanitabarrett @hannalaakso any thoughts?